tests/manager/api/admin/test_routes.py

```diff
@@ -373,6 +384,28 @@ def test_suppress(self, fixture: AdminRouteFixture):
             http_method="POST",
         )
 
+    def test_suppression_delete(self, fixture: AdminRouteFixture):
+        url = "/admin/works/<identifier_type>/an/identifier/suppression"
+        fixture.assert_authenticated_request_calls(
+            url,
+            fixture.controller.unsuppress,  # type: ignore
+            "<identifier_type>",
+            "an/identifier",
+            http_method="DELETE",
+        )
+
+        # TODO: when /suppress end point is removed (currently deprecated we'll want to remove the following two lines
+        # since these tests will start failing here.
+        url = "/admin/works/<identifier_type>/an/identifier/suppress"
+        fixture.assert_authenticated_request_calls(
+            url,
+            fixture.controller.unsuppress,  # type: ignore
+            "<identifier_type>",
+            "an/identifier",
+            http_method="DELETE",
```

Comment:
The `/suppress` endpoint doesn't have a "DELETE" method. Copypasta? Or did you mean to POST to the `/unsuppress` endpoint here? There's a test below that does that.

Reply:
Good catch - yes.  Copypasta indeed.  Fixed.